packages/core/src/actions/contracts/readContracts.ts

```diff
@@ -89,6 +89,9 @@ export async function readContracts<
             logWarn(result.reason.message)
             throw result.reason
           }
+          if (result.reason.message) {
```

Comment:
This was probably annoying to debug! Can we add a check against `ProviderChainsNotFound` here too? Better to opt in to what we are logging out.